source/common/http/codec_client.cc

```diff
@@ -140,7 +140,8 @@ CodecClientProd::CodecClientProd(Type type, Network::ClientConnectionPtr&& conne
     : CodecClient(type, std::move(connection), host, dispatcher) {
   switch (type) {
   case Type::HTTP1: {
-    codec_ = std::make_unique<Http1::ClientConnectionImpl>(*connection_, *this);
+    codec_ = std::make_unique<Http1::ClientConnectionImpl>(*connection_,
```

Comment:
Mind updating docs/root/configuration/http_conn_man/stats.rst ?

Each codec has the option of adding per-codec statistics. Currently only http2 has codec stats. needs updating, and a section on Http1 codec stats?

Reply:
Fixed. Thanks!